apps/web/app/(app)/environments/[environmentId]/surveys/components/SurveyDropdownMenu.tsx

```diff
@@ -49,11 +53,11 @@ export const SurveyDropDownMenu = ({
 
   const surveyUrl = useMemo(() => webAppUrl + "/s/" + survey.id, [survey.id, webAppUrl]);
 
-  const handleDeleteSurvey = async (survey: TSurvey) => {
+  const handleDeleteSurvey = async (surveyId: string) => {
     setLoading(true);
     try {
-      await deleteSurveyAction({ surveyId: survey.id });
-      deleteSurvey(survey.id);
+      await deleteSurveyAction({ surveyId });
+      deleteSurvey(surveyId);
```

Comment:
What is this doing? Again could maybe a router.refresh be a better choice here instead of prop drilling?

Reply:
Same as above